docs/pages/platform_guides/environments.mdx

```diff
@@ -0,0 +1,117 @@
+import Documentation from "df/docs/layouts/documentation";
+
+export const meta = {
+  title: "Configure environments",
+  __filename
+};
+
+export default ({ children }) => <Documentation {...meta}>{children}</Documentation>;
+
+## Introduction
+
+Environments can be used to configure different variations of your schedules. For each environment
+you define, Dataform will run all of the schedules defined in that environment, using that environment's
```

Comment:
If I understand things correctly, users do not define schedules in environment, right? I would have imagined that "Dataform will run all of the schedules defined in your project" was more explicit in that case. Am I missing anything?

I would emphasize on this sentence as well since the behaviour is quite important. With bolded text or else